### PR TITLE
Add _opam + bytes compatibility

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

File: _opam [hunk 1]
```diff
@@ -0,0 +1,5 @@
+depexts: [
```
Comment (ygrek) [inline _opam:1]: Why is this needed? It is already available in "official" opam file.

Comment (Chris00) [inline _opam:1]: That way, you simply do `oasis2opam <tarball URL>` and the opam metadata is automatically generated from your `_oasis` file (which has no provision for `depexts`).  This ensures that the OPAM metadata is kept in sync with the file that drives your build.

**Comment by ygrek** [comment]
Thank you!
